src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
```

Comment:
S2L is a [leaky abstraction](http://www.joelonsoftware.com/articles/LeakyAbstractions.html) on top of Selenium and many/most people working with the former for a longer time need to understand something about the latter too. Using same terms makes understanding concepts a lot easier.

The above said, it seems Selenium also uses just term capabilities when talking about the current capabilities. In that regard `get_browser_capabilities` is a fine name. It would probably then be a good idea to also access `self._current_browser().capabilities` in the code below and not its alias `self._current_browser().desired_capabilities`.